src/NUnitEngine/nunit.engine/Services/ResultWriters/XmlTransformResultWriter.cs

```diff
@@ -90,7 +90,7 @@ public void WriteResultFile(XmlNode result, TextWriter writer)
 
         public void WriteResultFile(XmlNode result, string outputPath)
         {
-            using (var xmlWriter = new XmlTextWriter(outputPath, Encoding.Default))
+            using (var xmlWriter = new XmlTextWriter(outputPath, Encoding.UTF8))
```

Comment:
What about @jnm2's comments about using `XmlWriter.Create(new StreamWriter(outputPath))` to ensure no BOM https://github.com/nunit/nunit-console/issues/181#issuecomment-282442507. See also the note on MSDN about `XmlTextWriter` that @jnm2 linked to.

> Starting with the .NET Framework 2.0, we recommend that you create XmlWriter instances by using the XmlWriter.Create method and the XmlWriterSettings class to take advantage of new functionality.

Reply:
Updated to `XmlWriter.Create`. `UTF-8` should be default according to API.